Computer Oriented Statistical Methods/Simpson's_1-3rd_rule.c

```diff
@@ -1,23 +1,25 @@
 #include<conio.h>
```

Comment:
[Why the use of conio.h is not good habit of programming - Stackoverflow](https://stackoverflow.com/questions/19926759/why-the-use-of-conio-h-is-not-good-habit-of-programming)